Formula/solc-select.rb

```diff
@@ -0,0 +1,32 @@
+class SolcSelect < Formula
+  include Language::Python::Virtualenv
+
+  desc "Manage multiple Solidity compiler versions"
+  homepage "https://github.com/crytic/solc-select"
+  url "https://files.pythonhosted.org/packages/48/d6/35f72b61b89c087e7b886ff6511deb6d3193db2ffacdcf03827373e5e312/solc-select-0.2.1.tar.gz"
+  sha256 "e956b04dc7df2209d1fb3b82e2bb62f8e730bb554c4d7f958a14ff2fb2f37212"
+  license "AGPL-3.0-only"
+  head "https://github.com/crytic/solc-select.git", branch: "dev"
+
+  depends_on "python@3.10"
+
+  resource "pysha3" do
+    url "https://files.pythonhosted.org/packages/73/bf/978d424ac6c9076d73b8fdc8ab8ad46f98af0c34669d736b1d83c758afee/pysha3-1.0.2.tar.gz"
+    sha256 "fe988e73f2ce6d947220624f04d467faf05f1bbdbc64b0a201296bb3af92739e"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system "solc-select", "install", "0.5.7"
```

Comment:
Where do these installed compilers go?

Reply:
Hi @carlocab! `solc-select` maintains the installed compilers and configuration in `~/.solc-select` ([code here](https://github.com/crytic/solc-select/blob/dev/solc_select/constants.py#L9)). From the cookbook, `The environment variable HOME is set to [testpath](https://rubydoc.brew.sh/Formula#testpath-instance_method) within the [test do](https://rubydoc.brew.sh/Formula#test-class_method) block.` so I believe this shouldn't pollute the system where tests are ran.